sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -55,6 +55,7 @@ statement
         rowFormat?  createFileFormat? locationSpec?
         (TBLPROPERTIES tablePropertyList)?
         (AS? query)?                                                   #createTable
+    | createTableHeader LIKE source=tableIdentifier                    #createTableLike
```

Comment:
I just found that in `HiveCompatibilitySuite`, one test `create_like_tbl_props` uses the syntax:

    CREATE TABLE test_table LIKE src TBLPROPERTIES('key'='value')

So do we want to support it? Or just disable this test? @andrewor14 @hvanhovell